src/html/js/user.js

```diff
@@ -765,8 +765,8 @@ function closeHelpConfirm(f) {
 }
 
 function viewInfoDisclosureDetail(type){
-    // dummy function
-    console.log(type);
+    $("#viewInfoDisclosureDetail #pageTitle").html(i18next.t("profile." + type));
```

Comment:
Any reason why you cannot use $('#pageTitle') which is supposed to be unique, too?